src/pages/item/item-actions/ItemActions.tsx

```diff
@@ -21,29 +31,98 @@ export default function ItemActions(props: { itemId: string | number }) {
 				"Can't load Downstream References - data still loading or failed to do so."
 			);
 		}
+		
+	};
+
+	//get all downstreamRefs and associations and then call in the miro.api a method which creates all connectors
+	// { downstreamRefs: [number], associations: [{associationId: number, targetItemId: number}] }
+	React.useEffect(() => {
+		async function fetchData() {
+			if (data && (data.downstreamReferences.length || data.outgoingAssociations.length)) {
+			var downstreamRefs: React.SetStateAction<number[]> = [];
+			var associations: ((prevState: never[]) => never[]) | { associationId: number; targetItemId: number; }[] = [];
```

Comment:
What's the `prevState` about ? 🤔

Reply:
I think it was a quick fix but I removed it and it still works